packages/core/test/install/hardLinkLocalPackages.ts

```diff
@@ -0,0 +1,237 @@
+import path from 'path'
+import assertProject from '@pnpm/assert-project'
+import { MutatedProject, mutateModules } from '@pnpm/core'
+import { preparePackages } from '@pnpm/prepare'
+import rimraf from '@zkochan/rimraf'
+import pathExists from 'path-exists'
+import { testDefaults } from '../utils'
+
+test('hard link local packages', async () => {
```

Comment:
I already test that `is-positive` is not present in the modules of project-1 but present when project-1 is a  dependency of project-2

Reply:
Generally, a local package declares `is-positive` as `peerDependency`, it would delcare `is-positive` as `devDependency` as well. I see `is-positive` is be declared as a peerDep only in project1. I wonder it is same or it is subtle different.

Reply:
It is the same. We just test that the package is "copied", so it has its own set of dependencies. There are other tests in the repo that cover other aspects.